sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Specify the release date by replace `Unreleased`, the format is `yyyy-mm-dd`.

Reply:
added a release date.